test/UnitTestHelpers.hs

```diff
@@ -398,4 +390,12 @@ instance MonadClientAtomic CliMock where
 executorCli :: CliMock a -> CliState -> IO (a, CliState)
 executorCli = runStateT . runCliMock
 
+-- | Transform AttrString type to Text
+--
+attrStringToText :: AttrString -> Text.Text
+attrStringToText  = Text.pack . map Color.charFromW32
```

Comment:
How about using this one (or the other?) in other places in the code? `git grep` would show you where to use.

Reply:
Did you miss this comment or you'd rather leave the refactoring (using `attrStringToText` elsewhere) for later?